platform/ui/src/components/TableHead/TableHead.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const TableHead = ({ children, className, style }) => {
+  return (
+    <div
+      className={classnames(
+        'bg-custom-navy border-b border-custom-violetPale flex font-bold pr-2',
+        className
+      )}
+      style={style}
+    >
+      {React.cloneElement(children, {
+        isTableHead: true,
```

Comment:
This assumes TableHead receives a children of a specific component type. Should/can we check for this?